user-manual/backlog/backlog.rst

```diff
@@ -46,6 +46,44 @@ Searching for transfers and files
       :width: 80%
       :alt: Archivematica backlog tab showing all transfers in the backlog
 
+#. To change the columns that are visible in the tabular list of backlogged transfers,
+   click on the Select columns icon at the bottom of the table.
+
+#. This will present a list of the columns that are available for viewing in the
```

Comment:
I would combine this point with the previous one, as they're really the same step.

Reply:
Good point. Done.